automatic/glmixer/tools/chocolateyInstall.ps1

```diff
@@ -0,0 +1,19 @@
+﻿$ErrorActionPreference = 'Stop'
+
+$packageArgs = @{
+  packageName            = 'glmixer'
+  fileType               = 'exe'
+  url                    = 'https://sourceforge.net/projects/glmixer/files/Windows%20Binary/GLMixer_1.5-1329_Windows.exe'
```

Comment:
Odd, I was sure I had used that way in the past for the djv package (before moving it to an embedded package)